example_profile.ini

```diff
@@ -0,0 +1,19 @@
+; this is an example profile configuration for rolling pin
+; technically any value in the example.ini file can be used here
+; but it is mostly for overriding and defaulting more execution values
+
+[deploy]
+; this sets a default set of hosts to apply to (default for the '-h' argument)
+hosts = apps
+
+; this sets a default set of components to deploy (default for the '-c' argument)
+components = rollingpin
+
+; this sets the default set of hosts to restart (default for the '-r' arguments)
+restarts = all
```

Comment:
I am not a big fan of these configuration names, so please suggest better options...